extensions/amp-iframe/0.1/validator-amp-iframe.protoascii

```diff
@@ -72,6 +72,8 @@ tags: {  # <amp-iframe>
     name: "srcdoc"
     mandatory_oneof: "['src', 'srcdoc']"
   }
+  # <amp-bind>
+  attrs: { name: "[src]" }
```

Comment:
If the comment re amp-brightcove is true, we still end up with the following here:
<amp-iframe ... srcdoc="..." [src]="...">

This would be considered valid, despite the fact that it results in an amp-iframe tag with both `src` and `srcdoc` which is considered invalid (validation requires *exactly* one of these two attributes in this case).